src/buttons.css

```diff
@@ -26,6 +26,7 @@
  * </div>
  */
 .btn {
+  display: inline;
```

Comment:
I think the browser default style is `inline-block`, so if you're removing the `display: inline-flex` declaration, I don't think you have to replace it with `display: inline`.